app/Http/Controllers/AnnouncementController.php

```diff
@@ -0,0 +1,17 @@
+<?php
+namespace App\Http\Controllers;
+
+use Illuminate\Routing\Controller as BaseController;
+use App\Announcement as Announcement;
+use App\ViewModel as ViewModel;
+
+class AnnouncementController extends BaseController
+{
+    public function index()
+    {
+        $vm = new ViewModel();
```

Comment:
Same here. Remove the `ViewModel` and pass the announcement to the view. You will need to get the details for a specific announcement here, instead of returning all of them. What we need to do is get the announcement that doesn't have an ama record created for it. This means, the Ama is announced, but the responses are not published yet.
Do this:
`$amaAnnouncement = Announcement::doesntHave('amas')->orderBy('created_at', 'desc')->first()`. This will get the latest Ama announcement that doesn't have a related ama record.